From b642542c396a9c3839502fddf92d3e794a2c8b7b Mon Sep 17 00:00:00 2001
From: Jason Rogena <j.rogena@cgiar.org>
Date: Wed, 28 May 2014 19:53:41 +0300
Subject: [PATCH 175/213] Version 0.9.6: Database Update for Cleaning Purposes,
 Removal of SMS Support (for now)

The database version updated so as to clean data from the farmer's
SQLite databases

SMS Support temporarily removed from the Application
---
 MistroFarmer/src/main/AndroidManifest.xml                             | 4 ++--
 .../src/main/java/org/cgiar/ilri/mistro/farmer/AddEventActivity.java  | 2 +-
 .../java/org/cgiar/ilri/mistro/farmer/AddMilkProductionActivity.java  | 2 +-
 .../java/org/cgiar/ilri/mistro/farmer/CowRegistrationActivity.java    | 2 +-
 .../main/java/org/cgiar/ilri/mistro/farmer/EventsHistoryActivity.java | 2 +-
 .../java/org/cgiar/ilri/mistro/farmer/FarmerRegistrationActivity.java | 2 +-
 .../src/main/java/org/cgiar/ilri/mistro/farmer/LandingActivity.java   | 2 +-
 MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/MainMenu.java | 2 +-
 .../org/cgiar/ilri/mistro/farmer/MilkProcutionHistoryActivity.java    | 2 +-
 .../java/org/cgiar/ilri/mistro/farmer/MilkProductionActivity.java     | 2 +-
 .../main/java/org/cgiar/ilri/mistro/farmer/backend/DataHandler.java   | 4 +++-
 .../org/cgiar/ilri/mistro/farmer/backend/database/DatabaseHelper.java | 2 +-
 12 files changed, 15 insertions(+), 13 deletions(-)

diff --git a/MistroFarmer/src/main/AndroidManifest.xml b/MistroFarmer/src/main/AndroidManifest.xml
index 1e3ef49..95d499e 100644
--- a/MistroFarmer/src/main/AndroidManifest.xml
+++ b/MistroFarmer/src/main/AndroidManifest.xml
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="org.cgiar.ilri.mistro.farmer"
-    android:versionCode="5"
-    android:versionName="0.9.5" >
+    android:versionCode="6"
+    android:versionName="0.9.6" >
 
     <uses-sdk
         android:minSdkVersion="7"
diff --git a/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/AddEventActivity.java b/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/AddEventActivity.java
index 9076111..3b2e1c7 100644
--- a/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/AddEventActivity.java
+++ b/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/AddEventActivity.java
@@ -114,7 +114,7 @@ public class AddEventActivity extends SherlockActivity implements MistroActivity
     {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_add_event);
-        DataHandler.requestPermissionToUseSMS(this);
+        //DataHandler.requestPermissionToUseSMS(this);
 
         cacheData = true;
 
diff --git a/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/AddMilkProductionActivity.java b/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/AddMilkProductionActivity.java
index 095b4be..0beb80c 100644
--- a/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/AddMilkProductionActivity.java
+++ b/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/AddMilkProductionActivity.java
@@ -83,7 +83,7 @@ public class AddMilkProductionActivity extends SherlockActivity implements Mistr
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_add_milk_production);
-        DataHandler.requestPermissionToUseSMS(this);
+        //DataHandler.requestPermissionToUseSMS(this);
 
         cacheData = true;
         milkQuantityFine = false;
diff --git a/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/CowRegistrationActivity.java b/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/CowRegistrationActivity.java
index 3d23263..1bc9866 100644
--- a/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/CowRegistrationActivity.java
+++ b/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/CowRegistrationActivity.java
@@ -135,7 +135,7 @@ public class CowRegistrationActivity extends SherlockActivity implements MistroA
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_cow_registration);
-        DataHandler.requestPermissionToUseSMS(this);
+        //DataHandler.requestPermissionToUseSMS(this);
 
         cacheData = true;
 
diff --git a/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/EventsHistoryActivity.java b/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/EventsHistoryActivity.java
index 0e3850f..6c602b4 100644
--- a/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/EventsHistoryActivity.java
+++ b/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/EventsHistoryActivity.java
@@ -61,7 +61,7 @@ public class EventsHistoryActivity extends SherlockActivity implements MistroAct
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_events_history);
-        DataHandler.requestPermissionToUseSMS(this);
+        //DataHandler.requestPermissionToUseSMS(this);
 
         eventHistoryIDs = new ArrayList<String>();
         metrics=new DisplayMetrics();
diff --git a/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/FarmerRegistrationActivity.java b/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/FarmerRegistrationActivity.java
index 07cc552..b663fce 100644
--- a/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/FarmerRegistrationActivity.java
+++ b/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/FarmerRegistrationActivity.java
@@ -66,7 +66,7 @@ public class FarmerRegistrationActivity extends SherlockActivity implements Mist
     {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_farmer_registration);
-        DataHandler.requestPermissionToUseSMS(this);
+        //DataHandler.requestPermissionToUseSMS(this);
 
         //init child views
         cacheData = true;
diff --git a/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/LandingActivity.java b/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/LandingActivity.java
index a9b063c..2716a0f 100644
--- a/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/LandingActivity.java
+++ b/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/LandingActivity.java
@@ -59,7 +59,7 @@ public class LandingActivity extends SherlockActivity implements MistroActivity,
     {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_landing);
-        DataHandler.requestPermissionToUseSMS(this);
+        //DataHandler.requestPermissionToUseSMS(this);
 
         //initialize child views
         loginButton=(Button)this.findViewById(R.id.login_button);
diff --git a/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/MainMenu.java b/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/MainMenu.java
index 509d718..52261b0 100644
--- a/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/MainMenu.java
+++ b/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/MainMenu.java
@@ -46,7 +46,7 @@ public class MainMenu extends SherlockActivity implements MistroActivity, View.O
     {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main_menu);
-        DataHandler.requestPermissionToUseSMS(this);
+        //DataHandler.requestPermissionToUseSMS(this);
 
         milkProductionB=(Button)this.findViewById(R.id.milk_production_b);
         milkProductionB.setOnClickListener(this);
diff --git a/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/MilkProcutionHistoryActivity.java b/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/MilkProcutionHistoryActivity.java
index 9fb3581..48e3c8f 100644
--- a/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/MilkProcutionHistoryActivity.java
+++ b/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/MilkProcutionHistoryActivity.java
@@ -69,7 +69,7 @@ public class MilkProcutionHistoryActivity extends SherlockActivity implements Mi
     {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_milk_production_history);
-        DataHandler.requestPermissionToUseSMS(this);
+        //DataHandler.requestPermissionToUseSMS(this);
 
         productionHistoryIDs=new ArrayList<String>();
         metrics=new DisplayMetrics();
diff --git a/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/MilkProductionActivity.java b/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/MilkProductionActivity.java
index 8a69068..7ae5091 100644
--- a/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/MilkProductionActivity.java
+++ b/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/MilkProductionActivity.java
@@ -77,7 +77,7 @@ public class MilkProductionActivity extends SherlockActivity implements MistroAc
     {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_milk_production);
-        DataHandler.requestPermissionToUseSMS(this);
+        //DataHandler.requestPermissionToUseSMS(this);
 
         addProductionB=(Button)this.findViewById(R.id.add_production_b);
         addProductionB.setOnClickListener(this);
diff --git a/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/backend/DataHandler.java b/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/backend/DataHandler.java
index b1df021..9ddd827 100644
--- a/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/backend/DataHandler.java
+++ b/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/backend/DataHandler.java
@@ -63,6 +63,7 @@ public class DataHandler
     public static final String CODE_USER_NOT_AUTHENTICATED="43322";
     public static final String CODE_SIM_CARD_REGISTERED="83242";
     public static final String CODE_NUMBER_IN_USE="032443";
+    public static final String NO_CONNECTION = "43387";
 
     private static final String TAG="DataHandler";
     private static final long SMS_RESPONSE_TIMEOUT = 300000;
@@ -194,7 +195,8 @@ public class DataHandler
             response = sendDataUsingHttpConnection(context, jsonString, appendedURL);
         }
         else{
-            response = sendDataUsingSMS(context, jsonString, appendedURL, waitForResponse);
+            //response = sendDataUsingSMS(context, jsonString, appendedURL, waitForResponse);
+            return null;
         }
         return response;
     }
diff --git a/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/backend/database/DatabaseHelper.java b/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/backend/database/DatabaseHelper.java
index 265232e..690cc05 100644
--- a/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/backend/database/DatabaseHelper.java
+++ b/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/backend/database/DatabaseHelper.java
@@ -18,7 +18,7 @@ import java.io.Serializable;
 public class DatabaseHelper extends SQLiteOpenHelper implements Serializable{
 
     public static final String DB_NAME = "ngombe_planner";
-    public static final int DB_VERSION = 9;
+    public static final int DB_VERSION = 10;
     public static final String TABLE_FARMER = "farmer";
     public static final String TABLE_COW = "cow";
     public static final String TABLE_EVENT = "event";
-- 
2.7.1

